dae/dae/backends/impala/loader.py

```diff
@@ -0,0 +1,93 @@
+import os
+import configparser
+import glob
+import pandas as pd
+import pyarrow.parquet as pq
+from dae.backends.raw.loader import VariantsLoader, TransmissionType
+from dae.backends.impala.serializers import ParquetSerializer
+from dae.variants.variant import SummaryVariantFactory
+
+
+class ParquetLoader(VariantsLoader):
+    def __init__(self, families,
+                 partition_config_file, partitions_to_read=[],
+                 transmission_type=TransmissionType.transmitted):
+        super(ParquetLoader, self).__init__(
+            families=families,
+            transmission_type=transmission_type)
+        self._read_partition_config(partition_config_file)
+        self.root_dir = os.path.dirname(partition_config_file)
+        self.partitions_to_read = partitions_to_read
+
+    def _read_partition_config(self, config_file):
+        self._config = configparser.ConfigParser()
+        self._config.read(config_file)
+        assert 'region_bin' in self._config
+        self._file_depth = 1
+        if 'family_bin' in self._config:
+            self._file_depth += 1
+        if 'coding_bin' in self._config:
+            self._file_depth += 1
+        if 'frequency_bin' in self._config:
+            self._file_depth += 1
+
+    def _get_filenames_to_open(self):
+        if self.partitions_to_read == []:
+            glob_path = '/'.join(['*'] * (self._file_depth + 1))
+            return glob.glob(os.path.join(self.root_dir, glob_path))
+        else:
+            filenames = []
+            for part in self.partitions_to_read:
+                filenames += glob.glob(os.path.join(self.root_dir, part))
+
+            return filenames
+
+    def _parquet_file_iterator(self, filename):
+        table = pq.read_table(filename)
+        df = table.to_pandas()
```

Comment:
this reads all data from parquet file into memory... strongly prefer to read by row groups and iterator over row groups keeping only one row group into memory...